### PR TITLE
v0

### PR DESCRIPTION
**Pull request by rohit22**
Very basic version of the lambda function and spectrum tables so that Mitchelle's team can query and look at the data. 

This will definitely lead to more work once they start looking at the data and determine the important information and their usecase. 

I haven't setup the deployment stuff for the lambda. That would be my next task after I let Mitchelle access the data. The next version could even be built in golang for the ease of implementation.

To Do:

1. Add deployment script for the python function.
2. Automate addition of partitions every day.

### CODE REVIEW

**Comment by rohit22** [comment]
Testing webhook.

File: lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  records[event_type].append(data_to_s3)
+  print(len(data_to_s3.split('\t')))
+  print(data_to_s3)
+  
+ s3 = boto3.resource("s3") 
+ for event_t in records.keys():
+  s3_path = event_t + "/" + "activity_date="+date_part + "/" + filename + "-" + str(d.microsecond) + ".tsv" 
+  print("writing " + event_t + " records. Total of: " + str(len(records[event_t])))
+  print("keys: " + record_keys[event_t])
+  s3.Bucket(bucket_name).put_object(Key=s3_path, Body='\n'.join(records[event_t]))
```
Comment (davidtime) [inline lambda_function.py]: would it be hard to gz the data?

Comment (rohit22) [reply lambda_function.py]: Shouldn't be hard. Let me try it. 

For backloading, I am not yet sure. I will discuss with Michelle.

**Comment by dveluvolu** [comment]
lgtm. 
minor: Any specific reason for not reusing an existing bucket (tunein-redshift-spectrum-*) for this scraper?

**Pull request merged**